doc/howto/network_create.md

```diff
@@ -1,15 +1,77 @@
 # How to create and configure a network
 
-The desired type can be specified using the `--type` argument, e.g.
+To create and configure a managed network, use the `lxc network` command and its subcommands.
+Append `--help` to any command to see more information about its usage and available flags.
+
+(network-types)=
+## Network types
+
+The following network types are available:
+
+```{list-table}
```

Comment:
Are these still clickable in github review, or is this perhaps another step towards removing/renaming the docs so they don't render in github at all?

Reply:
Its fine, but as the documentation is getting ever increasingly unusable in the github viewport, should we reconsider naming these files in such a way as to not generate a github view at all (so they are just displayed as raw text)?

Reply:
Well, they are still markdown files. And we have a readme in the doc folder that explains that the GitHub rendering is very basic, so I think that should be sufficient. But I don't have any kind of data if people actually read the docs on GitHub instead of on the website?